### ISSUE TITLE
Avoid using /usr/bin/tar on Debian machines

### ISSUE THREAD

**Issue by kevinushey** [rstudio/r-builds#198]
This is a follow-up from https://github.com/rstudio/renv/issues/1794.

For the builds of R published on https://docs.posit.co/resources/install-r, it looks like R is configured to use system installations of `tar` and `sed` as so:

```
$ cat Renviron | grep 'SED\|TAR'
SED=${SED-'/usr/bin/sed'}
TAR=${TAR-'/usr/bin/tar'}
```

This can be problematic in rare scenarios, as `/usr/bin/sed` and `/usr/bin/tar` are not actually provided by any `apt` packages. So, if those programs are not available for some reason, it's not clear for the user on how to recover.

Some other similar instances:

```
$ cat Renviron | grep /usr/bin/
R_PRINTCMD=${R_PRINTCMD-'/usr/bin/lpr'}
R_TEXI2DVICMD=${R_TEXI2DVICMD-${TEXI2DVI-'/usr/bin/texi2dvi'}}
R_GZIPCMD=${R_GZIPCMD-'/usr/bin/gzip'}
R_UNZIPCMD=${R_UNZIPCMD-'/usr/bin/unzip'}
R_ZIPCMD=${R_ZIPCMD-'/usr/bin/zip'}
R_BZIPCMD=${R_BZIPCMD-'/usr/bin/bzip2'}
PAGER=${PAGER-'/usr/bin/pager'}
R_PDFVIEWER=${R_PDFVIEWER-'/usr/bin/xdg-open'}
SED=${SED-'/usr/bin/sed'}
TAR=${TAR-'/usr/bin/tar'}
```

In other words, for each of these programs, if a variant exists in `/bin`, I think that version should be preferred.

**Comment by gaborcsardi** [comment]
Yeah, it is the upgrades that cause issues, the symlinks are missing on upgraded systems.

**Comment by kevinushey** [comment]
Thanks; I wasn't aware of that! For what it's worth:

```
$ dpkg -L tar | grep /bin/tar
/bin/tar

$ dpkg -L sed | grep /bin/sed
/bin/sed
```

which I think gives some (small) amount of motivation to prefer `/bin/tar` over `/usr/bin/tar`.

**Comment by gaborcsardi** [comment]
We had this in rig some time ago, for `sed` (https://github.com/r-lib/rig/issues/119#issuecomment-1644040882), apparently it also applies to `tar`.
It would be nice to fix it properly.
